src/domain/event/eventOccurrence.module.scss

```diff
@@ -8,16 +8,46 @@
     width: 100%;
   }
 }
-
-table {
-  border-collapse: collapse;
-}
-
 tr {
   border-bottom: 2px solid $hel-light;
 }
 
-th,
-td {
-  padding: $basePadding 0;
+@include respond-above(md) {
+  table {
+    border-collapse: collapse;
+  }
+  th,
+  td {
+    padding: $basePadding 0;
+  }
+}
+@include respond-below(md) {
+  tr {
+    display: grid;
+    grid-template-rows: auto;
+    grid-template-columns: auto;
+    grid-row-gap: $basePadding;
+    padding: 1rem 0;
+  }
+  td.occurrenceFree {
+    grid-column: 4;
+    grid-row: 1 / span 2;
+    justify-self: end;
+    align-self: center;
+  }
+  td.occurrenceVenue {
+    grid-row: 1;
+    grid-column: 1 / span 3;
+  }
+  td.occurrenceDate {
+    grid-row: 2;
+    grid-column: 1 / span 2;
+  }
+  td.occurrenceTime {
+    grid-row: 2;
+    grid-column: 2 / span 2;
+  }
```

Comment:
Date and time starts to overlap at around 400px width. Is it something we need to consider/support?
![localhost_3000_fi_event_RXZlbnROb2RlOjE=](https://user-images.githubusercontent.com/969494/75709579-9d931500-5ccb-11ea-9ce2-46272d51f204.png)

Reply:
Do any device exist that have a 400px screen width?

Reply:
Let me see if we can find a better solution

Reply:
Does it work better now?